src/23-2017-q1-update/index.md

```diff
@@ -0,0 +1,62 @@
+---
+# this is the final blog post's id (used in the directory)
+# this id is part of the url, and should only contain:
+#   letters, numbers, dashes.
+id: 2017-q1-update
+
+breadcrumbs:
+  - {name: "2017-q1-update", link: "./" }
+
+# the date here should be set to the final publication date,
+# on the day it is published.
+date: 2017-01-25
+
+# this is the Title
+title: IPFS Project Management Update for Q1 2017
+
+# this is the name of the main author(s)
+author: Richard Littauer
+
+# technical details required for the software, don't change these.
+baseurl: ..
+template: tmpl/layouts/post.html
+collection: posts
+---
+
+# IPFS going into Q1 in 2017
+
+This year, IPFS is changing the way we do our sprints. This post covers some of the basic aspects of these changes. A longer post will be coming in February.
+
+In short, we've switched from weekly set-your-own-goal sprints focused on particular areas - js-ipfs, go-ipfs, Apps on IPFS, and so on - and we're moving, as a project, to more defined two week sessions where we work together on scrum-like projects.
```

Comment:
Previously we used a loosely organized approach where each team member would set their own goals for a sprint. We broke our work up under three headings: js-ipfs, go-ipfs or "Apps on IPFS".  Now we're moving to a scrum-like structure, where we form teams around targeted, two-week scrum-style Sprints.